x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/embeddings/CohereEmbeddingsTaskSettings.java

```diff
@@ -31,18 +35,17 @@
  * <p>
  * <a href="https://docs.cohere.com/reference/embed">See api docs for details.</a>
  * </p>
- *
- * @param inputType Specifies the type of input you're giving to the model
- * @param truncation Specifies how the API will handle inputs longer than the maximum token length
  */
-public record CohereEmbeddingsTaskSettings(@Nullable InputType inputType, @Nullable CohereTruncation truncation) implements TaskSettings {
+public class CohereEmbeddingsTaskSettings implements TaskSettings {
```

Comment:
I switched this to a class because I thought we'd need to do some `if-blocks` for the transport version. Turns out I don't think we need to but double check me here.